### PR TITLE
Fix Bayer notation parsing & matching regresion.

### PR DESCRIPTION
**Pull request by pirogronian**
Fixing #257 .
Added: fixing (hopely) #262 .

### CODE REVIEW

File: src/celutil/utf8.cpp [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (375gnu) [inline src/celutil/utf8.cpp]: extra new line

File: src/celutil/utf8.cpp [hunk 2]
```diff
@@ -941,10 +941,21 @@ static int findGreekNameIndexBySubstr(const std::string &s, int start, unsigned
...
 }
 
+static size_t greekChunkLength(const std::string& str)
+{
+    size_t sp = str.find_first_of(' ');
+    if (sp == std::string::npos)
+        sp = str.length();
+    if (str[sp - 1] > '0' && str[sp -1] < '4')
+        sp--;
+    else sp = std::string::npos;
```
Comment (375gnu) [inline src/celutil/utf8.cpp]: please move `sp = ` to a new line

**Comment by pirogronian** [comment]
In fact it seems to be replaced, but by abbreviation, not by Greek letter (!).

**Comment by 375gnu** [comment]
Please also fix the 2nd commit name: fox -> fix.

**Comment by 375gnu** [comment]
hi @pirogronian do you have any additional plans for this PR? i want to build a new executables today when you merge it

**Pull request merged**